src/typings.ts

```diff
@@ -113,7 +251,19 @@ export type AuthorizeUrlOptions = Omit<AuthorizeOptions, 'isNewTab'>;
 export type Mode = 'production' | 'staging' | 'develop' | 'local';
 export type InitOptions = Omit<Omit<Omit<AuthorizeOptions, 'forceLogout'>, 'codeChallenge'>, 'pkce'> &
   PrivateParams & {
+    /**
+     * Environment for the SDK to connect to, the SDK will connect to the Moneytree production server by default.
+     * - Moneytree clients should use `staging` for development as `develop` may contain unstable features.
+     * - `local` is only for SDK development as it has local dependencies.
```

Comment:
Maybe we should mention "development by Moneytree engineers" to be on the safe side? Just a minor suggestion.